concepts/Common/Integration Guides/50 Use MUI components in your DevExtreme application/50 Use MUI components in your DevExtreme application.md

```diff
@@ -0,0 +1,182 @@
+# Use MUI components in DevExtreme apps
+
+MUI is a popular line of React components and design tools. The design of MUI components follows Google's Material Design guidelines. Most MUI products are open-source, and do not require a commercial license.
+
+You don't have to use MUI to add a Material Design look and feel to your application. The DevExtreme library includes [a range of  Material Design themes](https://devexpress.github.io/ThemeBuilder/master/material/blue-light). However, if you want to use both MUI components and DevExtreme components, you can.
+
+You don't need extra configuration to run MUI components side by side with DevExtreme components. But if you nest components from different frameworks, you have to understand how they interact with one another.
+
+## Run the sample application
+
+The DevExtreme team created a single-page, ready-to-run Vite application that illustrates two common use cases. It contains an MUI component with DevExtreme children, and a DevExtreme component with MUI children. You can view the application's source code in the following GitHub repository: [https://github.com/DevExpress-Examples/devextreme-mui-integration](https://github.com/DevExpress-Examples/devextreme-mui-integration).
+
+Install the dependencies and run the app:
+
+    npm i; npm run dev
+
+![Sample application example](/images/MUI/sample-app.png)
+
+## DevExtreme components inside MUI components
+
+The `src/MUI-Form-with-DX-editors.tsx` file contains an MUI form with DevExtreme inputs.
+
+### Form context and hooks
+
+The `FormControl` MUI component is simple in comparison to its DevExtreme counterpart. Its purpose is to keep track of the form state. Inputs and labels can access this data with the `useFormControlContext` hook.
+
+Since DX inputs do not have access to the `useFormControlContext` hook out of the box, we need to create a wrapper function:
+
+    <!--tab: MUI-Form-with-DX-editors.tsx-->
+    // In accordance with MUI conventions, do not capitalize the name of the wrapper function
+    const dX_MUI_Form_Wrapper_HOC = (DXComponent: any, componentName: keyof FormData) => (props: any) => { 
+        const { onValueChange, ...restProps } = props;
+        const value = useFormControlContext()?.value as FormData;
+        return <DXComponent value={value[componentName]} onValueChange={onValueChange} {...restProps}></DXComponent>
+    }
+
+And wrap both of our non-MUI inputs:
+
+    <!--tab: MUI-Form-with-DX-editors.tsx-->
+    const DXTextBoxWrapper = dX_MUI_Form_Wrapper_HOC(DXTextBox, 'textBox')
+    const DXCheckBoxWrapper = dX_MUI_Form_Wrapper_HOC(DXCheckBox, 'checkBox')
+
+### State management
+
+React.js developers recommend the use of [controlled state management mode](https://reactjs.org/docs/forms.html#controlled-components) for web forms.
```

Comment:
Do you think we need a link for an old React documentation? I just recently removed all the links to this website from our docs. Please reconsider the usage of this link.